README.md

```diff
@@ -22,9 +22,12 @@ You can also tab complete remote names and branch names e.g.: `git pull or<tab>
 - `master` avoids breaking changes, maintaining v0.x.
   ( [README](https://github.com/dahlbyk/posh-git/blob/master/README.md)
   • [CHANGELOG](https://github.com/dahlbyk/posh-git/blob/master/CHANGELOG.md) )
+  - Supports PowerShell 3+
```

Comment:
Probably should be `Windows PowerShell 3+` to help make distinction between Windows PowerShell and PowerShell Core.